docs: Add docs for LinkFree CLI (#5027)
EddieHubCommunity/LinkFree: pages/docs/environments/linkfree-cli.mdx

import DocsLayout from "../../../components/layouts/DocsLayout.js";
import Link from "../../../components/Link";

## Quick demo 

![LinkFree CLI demo](https://user-images.githubusercontent.com/51878265/220884496-3025e260-bcfe-4abe-b639-b66592dd4038.gif)

## What is LinkFree CLI?

> **LinkFree CLI** is a command line tool that helps us create a JSON and update anexisting JSON file for your LinkFree profile. We can also give testimonials to other LinkFree profiles and add events.

Reviewer: ```suggestion
**LinkFree CLI** is a command line tool that helps us create a JSON and update an existing JSON file for your LinkFree profile. We can also give testimonials to other LinkFree profiles and add events.
```

## Installation and using the CLI

1. Make sure you have cloned the [LinkFree](https://github.com/EddieHubCommunity/LinkFree) repository and have installed the dependencies. Also, make sure you are in the root directory of the project.

2. To trigger the CLI, run the following command.

```bash
npm run cli
```

- When we run the above command it will run `npx linkfree-cli`. We use **npx** to run the CLI without installing it globally.

> 3. Then you will then prompted with several options. You can use the arrow keys to navigate and press enter to select the task you want to perform and then you will be prompted with the questions.

Reviewer: ```suggestion
3. You will then be prompted with several options. You can use the arrow keys to navigate and press enter to select the task you want to perform and then you will be prompted with the questions.
```

![LinkFree CLI main prompt](https://user-images.githubusercontent.com/51878265/220891756-10861090-e719-4687-b4fb-f7bef2826934.png)

> 4. Once you are done with the questions, it will create the JSON file depending on the task you selected. For example, if you selected `Create a LinkFree JSON file`, it will create a JSON file in the `public/data` directory. You will also get a message in the terminal. Something similar to the following image.

Reviewer 2: I think this should be `data/` directory?

```suggestion
4. Once you are done with the questions, it will create the JSON file depending on the task you selected. For example, if you selected `Create a LinkFree JSON file`, it will create a JSON file in the `data/` directory. You will also get a message in the terminal. Something similar to the following image.
```
